templates/service-template-aro-hcp.yml

```diff
@@ -0,0 +1,351 @@
+---
```

Comment:
the existing templates are openshift-centric, like this annotation. They will work for us in ROSA HCP.

I made a separate set of AROHCP templates for CS, too, since they are different enough.